### DISCUSSION TITLE
Quarkus 3.4.2 released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#36304] [Quarkus Blog/Website]
# Quarkus 3.4.2 released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-4-2-released/

**Comment by AndriiMalyna** [comment]


**Reply by gsmet** [reply]
@AndriiMalyna WDYM? The quarkus-maven-plugin is released with Quarkus and 3.4.2 is available: https://central.sonatype.com/artifact/io.quarkus.platform/quarkus-maven-plugin/versions .

Please give us more details if you are encountering issues. Thanks!

**Reply by AndriiMalyna** [reply]
it looks like my mistake,I looked here https://mvnrepository.com/artifact/io.quarkus/quarkus-maven-plugin

**Reply by gsmet** [reply]
Yeah, these websites always lag a bit as they need to index Maven Central. Sometimes it takes time for the artifacts to appear there (and it might not be fully consistent).
